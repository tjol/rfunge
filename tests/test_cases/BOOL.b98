"LOOB"4#@($:.a,f1+3O.e7A.deX.1N.07-fX.1#v)a,@
               >:#,_@#:a"unload error"a0<