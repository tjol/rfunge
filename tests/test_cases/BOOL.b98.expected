1112493900 
19 6 3 -2 -10 
